Source/WTF/wtf/PlatformMac.cmake

```diff
@@ -83,6 +83,8 @@ list(APPEND WTF_SOURCES
     cocoa/URLCocoa.mm
     cocoa/WorkQueueCocoa.cpp
 
+    darwin/LibraryPathDiagnostics.mm
```

Comment:
I _think_ this should be here, but I'm not 100% sure and haven't verified locally that this fixes a full (i.e. _not_ jsc-only) CMake build for macOS. I tried to test this with `build-webkit --cmake` or `build-jsc --cmake` but those don't appear to work. How can I test this configuration?

Reply:
If we build this file, why don't we call initializeLibraryPathDiagnostics?

Reply:
As I understand it, this file is not used in the JSC-only build, but is used for the full "normal" macOS build of the stack with CMake. (Which to be honest, I'm not entirely sure whether CMake builds of the whole stack are supported for macOS.) In that case, we _would_ compile that file and call the function.

Reply:
> How can I test this configuration?

`build-webkit --jsc-only [--cmakeargs=....]`